test/choosetests.jl

```diff
@@ -176,6 +178,33 @@ function choosetests(choices = [])
         filter!(x -> !(x in net_required_for), tests)
     end
 
+    if "stdlib" in skip_tests
+        filter!(x -> x != "stdlib", tests)
+    elseif "stdlib" in tests
+        filter!(x -> x != "stdlib", tests)
+        prepend!(tests, stdlibs)
+    end
+
+    if startswith(string(Sys.ARCH), "arm")
+        # Remove profile from default tests on ARM since it currently segfaults
+        # Allow explicitly adding it for testing
+        warn("Skipping Profile tests")
+        filter!(x -> (x != "Profile"), tests)
+    end
+
+    stdlib_dir = joinpath(@__DIR__, "..", "stdlib")
+    for stdlib in filter(x -> x in stdlibs, tests)
+        splice!(tests, findfirst(equalto(stdlib), tests))
+        dir = readdir(stdlib_dir)
+        test_file = joinpath(stdlib_dir, stdlib, "test", "runtests")
+        if isfile(test_file * ".jl")
+            unshift!(tests, test_file)
```

Comment:
Not a big problem, but it seems that this algorithms will reverse the order of the stdlib tests (this cries for a `replace!` standard algorithm a la #22324 ;-) )